docs/README.md

```diff
@@ -1,8 +1,85 @@
-Documentation
-====
+# Website
 
-All documentation for Arcus Background Jobs which are published via GitHub Pages with Jekyll.
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
 
-## Running Jekyll locally
+## Installation
 
-Running Jekyll locally to test your changes is super easy, learn more about it [here](https://jekyllrb.com/docs/#instructions).
\ No newline at end of file
+```console
+yarn install
+```
+
+## Local Development
+
+```console
+yarn start
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+## Versioning
+https://docusaurus.io/docs/versioning
+
+You can use the version script to create a new documentation version based on the latest content in the docs directory. That specific set of documentation will then be preserved and accessible even as the documentation in the docs directory changes moving forward.
```

Comment:
What does this mean? That the script will create a new 'snapshot' of the current `./docs` folder + that this 'snapshot' will be accessible even as the `./docs` folder changes?

Reply:
Yes, the `./docs` folder replaces the previous `./preview` folder. When developing locally or on a PR preview deploy you will see a `next` version in the list of versions in the dropdown see: https://deploy-preview-73--arcus-background-jobs-staging.netlify.app/next/
On production, this is currently disabled but is easily set inside the config file to include the next version as well.

If you want to change an already existing version eg `0.2`, you have to change the contents inside the `./versioned_docs/version-0.2`.
Running the script will take the content of the `./docs` folder and place it inside the `./versioned_docs/version-${your_version}` folder.